comptests/motr/test_motr_S3bench_DI_tests.py

```diff
@@ -0,0 +1,23 @@
+     return list_s3bench_DI_comb
+
+def main():
+    acsKey    = args.a
+    secretKey = args.s
+    endPoint  = args.e
+    for test in s3bench_DI_combinations():
+        command = "s3bench -region us-east-1 -accessKey {} -accessSecret {} -bucket test1 -endpoint {} -numClients {} -numSamples {} -objectSize {}".format(acsKey, secretKey, endPoint, test['noOfClients'], test['noOfSamples'], test['objectSize'])
+        os.system(command)
+
+if __name__ == '__main__':
+    main()
```

Comment:
Looks like we are planning to run it like a script? Why are we adding this to the test framework? Are there any related TCs in the Test JIRA? I don't see any test IDs.